src/registrar/templates/application_withdraw_confirmation.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block title %}Withdraw request for {{ domainapplication.requested_domain.name }}{% endblock %}
+{% load static url_helpers %}
+
+{% block content %}
+<div class="grid-col desktop:grid-offset-2 desktop:grid-col-8">
+ 
+
+<p><h1>Withdraw request for {{ domainapplication.requested_domain.name }}?</h1></p>
```

Comment:
Don't need to wrap the `h1` in a `p` tag (causing some typographic misalignment)

Reply:
fixed.

Reply:
Might be the committing issue you mentioned in slack, but this change still doesn't show up for me – i.e. the `h1` is still in a `p` tag